### ISSUE TITLE
use of pgup/pgdn keys on calendar views

### ISSUE THREAD

**Issue by kjburns** [kjburns/task-calendar#1]
PgUp/PgDn should accomplish exactly the same task as rolling the mouse wheel. There is some code written already in CalendarWidget::buildUI(), but it's not working, probably because the widget doesn't have keyboard focus. Maybe registering a listener on the app instead will do the trick.
